playbook/app/pb_kits/playbook/pb_star_rating/_star_rating.scss

```diff
@@ -2,61 +2,94 @@
 
-$pb_star_rating_number_width: 30px;
-$pb_star_rating_wrapper_width: 90px;
-$pb_star_rating_height: 26px;
 
-[class^=pb_star_rating_kit] {
-  position: relative;
-  display: inline-flex;
+[class*=pb_star_rating_kit] {
+  display: flex;
+  align-items: flex-end;
   justify-content: flex-start;
-  align-items: flex-start;
-
-  .pb_star_rating_number {
-    flex-shrink: 0;
-    flex-grow: 0;
-    flex-basis: $pb_star_rating_number_width;
-    width: $pb_star_rating_number_width;
-    min-height: $pb_star_rating_height;
-    padding: 0 $space-xs 0;
-    color: $text_lt_light;
-    font-size: $text-small;
+
+  path {
+    &.suble_star_dark {
+      fill: $text_dk_default; // Replace "red" with your desired color
+    }
+    &.suble_star_light {
+      fill: #242B42; // Replace "red" with your desired color
+    }
+    &.outline_star_dark {
+      stroke: $text_dk_lighter; // Replace "red" with your desired color
+    }
+    &.outline_star_light {
+      stroke: #C1CDD6; // Replace "red" with your desired color
+    }
+    &.empty_star_dark {
+      fill: $border_dark; // Replace "red" with your desired color
+    }
+    &.empty_star_light {
+      fill: #E4E8F0; // Replace "red" with your desired color
+    }
+  }
+
+  .number_rails_xs {
+    height: 22px !important;
+  }
+
+  .number_rails_sm {
+    height: 22px !important;
+  }
+
+  .number_rails_md {
+    height: 26px;
+  }
+
+  .number_rails_lg {
+    height: 38px;
+  }
+
+  .pb_star_rating_wrapper {
+    display: flex;
+  }
+
+  .pb_star_rating_number_xs {
+    max-height: 16px;
+  }
+
+  .pb_star_rating_number_sm {
+    max-height: 16px;
+  }
+
+  .pb_star_rating_number_md {
+    max-height: 32px;
+  }
+
+  .pb_star_rating_number_lg {
+    max-height: 56px;
+  }
+
+  .pb_star_xs {
     display: flex;
+    align-items: center;
     justify-content: center;
+    padding-right: 4px;
+  }
+
+  .pb_star_sm {
+    display: flex;
     align-items: center;
+    justify-content: center;
+    padding-right: 4px;
   }
 
-  .pb_star_rating_wrapper {
-    position: relative;
-    width: $pb_star_rating_wrapper_width;
-    flex-basis: $pb_star_rating_wrapper_width;
-    flex-shrink: 0;
-    flex-grow: 1;
-    min-height: $pb_star_rating_height;
-
-    .pb_star_rating_highlight {
-      display: flex;
-      position: absolute;
-      top: 5px;
-      left: 0;
-      z-index: 5;
-      color: $yellow;
-      line-height: $pb_star_rating_height;
-    }
-    .pb_star_rating_base {
-      display: flex;
-      position: absolute;
-      top: 5px;
-      left: 0;
-      z-index: 0;
-      color: rgba($slate, $opacity-4);
-      line-height: $pb_star_rating_height;
-    }
+  .pb_star_md {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-right: 6px;
   }
 
-  &[class*=_hide_rating] {
-    .pb_star_rating_number {
-      display: none;
-    }
+  .pb_star_lg {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-right: 12px;
   }
```

Comment:
At a minimum you should make this a mixin and DRY this up

Reply:
using flex kit allowed me to get rid of this code